### DISCUSSION TITLE
Problem Fixed, New Suggestion?

### DISCUSSION THREAD

**Discussion by dscarb21** [EmilioBarradas/tiktok-app-api#17] [Ideas]
Hello,
My previous problem has been fixed, and it works wonderfully. Thanks!

Is there any way that you can get an exact follower count? For example, tiktok says that the count is 70.3k, so it returns 70300, but obviously that doesn't take the tens or ones place into account. 

Just wondering if it's possible to count every user, or if TikTok doesn't publicly show that information as an easy to grab variable.

**Comment by EmilioBarradas** [comment]
TikTok doesn't publicly disclose the exact follower count, just the rounded count.

**Reply by dscarb21** [reply]
Any idea how sites like https://tiktokcounter.com/ display exact counts? 

I think I might try web scraping that actual site to get a precise number. Any specific tutorials that you recommend to get started on API making?

Thanks again, I'll shut up now lol
-David

**Reply by ReverseAndCode** [reply]
You can get the exact number of followers by parsing the JSON responses of the internal TikTok API (not the website).
